src/SixLabors.Fonts/GlyphMetrics.cs

```diff
@@ -194,107 +234,175 @@ internal FontRectangle GetBoundingBox(Vector2 origin, float scaledPointSize)
         /// <param name="location">The location.</param>
         /// <param name="options">The options used to influence the rendering of this glyph.</param>
         /// <exception cref="NotSupportedException">Too many control points</exception>
-        public void RenderTo(IGlyphRenderer surface, float pointSize, Vector2 location, TextOptions options)
+        internal void RenderTo(IGlyphRenderer surface, float pointSize, Vector2 location, TextOptions options)
```

Comment:
Making this internal stops users from attempting to render glyphs without an autogenerated run. I've streamlined the process to use `Font.GetGlyphs(CodePoint codePoint, ColorFontSupport support)` which pushes users through the appropriate pipeline.